test/linter/test-versions.ts

```diff
@@ -18,18 +18,8 @@ import {
 import bcd from '../../index.js';
 const { browsers } = bcd;
 
-const validBrowserVersions: { [browser: string]: string[] } = {};
-
-const VERSION_RANGE_BROWSERS: { [browser: string]: string[] } = {
-  chrome: ['≤15', '≤37'],
-  edge: ['≤18', '≤79'],
-  ie: ['≤6', '≤11'],
-  opera: ['≤12.1', '≤15'],
-  opera_android: ['≤12.1', '≤14'],
-  safari: ['≤4'],
-  safari_ios: ['≤3'],
-  webview_android: ['≤37'],
-};
+const twoYearsAgo = new Date();
```

Comment:
For reproducibility if we're bisecting a bug or something, ideally we should only depend on the state of the repo and not the current time. How about we use the most recent stable release as "today"? That way it will keep moving forward, but not change every time the script is run.

Reply:
I've tried implementing, this is what I came up with:

```js
const getRangeCutoffDate = (): Date => {
  let cutoff;
  for (const browserData of Object.values(browsers)) {
    for (const release of Object.values(browserData.releases)) {
      if (release.status === 'current' && release.release_date) {
        const releaseDate = new Date(release.release_date);
        if (!cutoff || releaseDate > cutoff) {
          cutoff = releaseDate;
        }
      }
    }
  }
  cutoff.setFullYear(cutoff.getFullYear() - 2);
  return cutoff;
};

const twoYearsAgo = getRangeCutoffDate();
```

Reply:
May I suggest doing something substantially dumber: hardcode a date then advance it periodically?

I suggest this mainly because I'll do anything to get out of doing date math (I look up today's date constantly!) and that might be a nice consideration for data contributors: there's some (semi-)fixed point in time that all things are before and after, rather than a rebase-to-rebase moving target.

(Also, I wonder how this interacts with mirroring. What happens when an unreflected version is on the valid side of the version range cutoff, but the mirrored version is on the invalid side of the cutoff?)